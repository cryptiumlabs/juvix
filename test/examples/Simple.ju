sig foo 0 : Int -> Int