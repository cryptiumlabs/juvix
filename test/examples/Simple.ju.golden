[Signature' (Sig' {signatureName = Sym "foo", signatureUsage = Just (Constant' (Number' (Integer'' 0 ()) ()) ()), signatureArrowType = Infix' (Inf' {infixLeft = Name' (Sym "Int" :| []) (), infixOp = Sym "->" :| [], infixRight = Name' (Sym "Int" :| []) (), annInf = ()}) (), signatureConstraints = [], annSig = ()}) ()]